src/components/admin/article/ArticleController.jsx

```diff
@@ -75,20 +86,29 @@ class ArticleController extends FalcorController {
       );
     }
 
-    Promise.all(updatePromises).then(() => {
-      // Reset state after save is done
-      this.safeSetState({
-        changed: false,
-        refresh: true,
-        authorsAdded: [],
-        authorsDeleted: {},
-        changesObject: { mainForm: false, authors: false },
-      });
-      // This is purely so the 'saved' message can be seen by the user for a second
-      setTimeout(() => {
-        this.safeSetState({ saving: false });
-      }, 1000);
+    if (!processedTags !== null) {
+      updatePromises.push(
+        this.falcorCall(
+          ['articles', 'bySlug', articleSlug, 'tags', 'updateTags'],
+          [falcorData.id, processedTags],
+          [['name'], ['slug']],
+        ),
+      );
+    }
+
+    await Promise.all(updatePromises);
+    // Reset state after save is done
+    this.safeSetState({
+      changed: false,
+      refresh: true,
+      authorsAdded: [],
```

Comment:
I think all these 3 keys below here are old remnants that no longer do anything (not introduced by you), while you're at it I think you can just remove them right?

Reply:
Yup